tools/xpathgrep.py

```diff
@@ -143,7 +142,7 @@ def iter_input(input, filename, parser, line_by_line):
                         yield et.ElementTree(et.fromstring(line, parser))
             else:
                 yield et.parse(input, parser)
-        except IOError:
+        except OSError:
```

Comment:
This looks like a functional change. (Same below.)

Reply:
This is a modification from [PEP 3151](https://peps.python.org/pep-3151/#confusing-set-of-os-related-exceptions), implemented in Python 3.3. It's only a naming change